incubator/hnc/pkg/kubectl/set.go

```diff
@@ -174,6 +190,7 @@ func normalizeStringArray(in []string) []string {
 func newSetCmd() *cobra.Command {
 	setCmd.Flags().Bool("root", false, "Turns namespace into root namespace")
 	setCmd.Flags().String("parent", "", "Parent namespace")
+	setCmd.Flags().Bool("allowCascadingDelete", false, "Allows cascading deletion of its self-serve subnamespaces. The values can only be true or false.")
```

Comment:
Is this necessary? Will there errors from internal library saying the flag is boolean if user enters a non boolean value?

Reply:
+1

Reply:
It's true that if the value is not true or false, it will say "Invalid argument/syntax" followed by the `-help` info.

This text will show up in the `-help` or when users input invalid arguments, so it may be helpful having it here.

Reply:
Personally I'd take that out - almost all `allow` flags are booleans, and I don't think we need to document them all every time.

Reply:
sg. The "value" part is taken out.